packages/offix-cache/src/createMutationOptions.ts

```diff
@@ -190,11 +190,22 @@ function addItemToQuery({ mutationName, updateQuery, idField = "id" }: CacheUpda
       if (result && operationData) {
         // FIXME deduplication should happen on subscriptions
         // We do that every time no matter if we have subscription
-        if (!result.find((item: any) => {
-          return item[idField] === operationData[idField];
-        })) {
-          result.push(operationData);
-        }
+          if(result.find){
+              const foundItem = !result.find(function (item) {
+                  return item[idField] === operationData_1[idField];
+              });
+              if (foundItem) {
+                  result.push(operationData_1);
+              }
+
+          } else if(result.items && result.items.find){
```

Comment:
assumes `items` exsit for Pagination and sync (we use the same name)

Reply:
What is the impact if pagination and sync is not wanted?

Reply:
Zero. This will just make it work (so this else will never appear).
This can be also used upstream. if you want to wrap your offline data into some container you just need to name variable `items`.

Let's make build pass and release it.

Reply:
Actually there is problem here. If someone will have named field `find` it will stop working for them. Chance for that is very low though.

I had much simpler implementation before that used `items` for checking but that can actually be field on the object.